src/main/java/uk/ac/ebi/subs/validator/coordinator/CoordinatorListener.java

```diff
@@ -245,11 +247,7 @@ public void processFileDeletionRequest(FileDeletedMessage fileDeletedMessage) {
     public void processSubmittableDeletion(StoredSubmittableDeleteMessage storedSubmittableDeleteMessage) {
         String submissionID = storedSubmittableDeleteMessage.getSubmissionId();
 
-        if (!fileValidationRequestHandler.handleFilesWhenSubmittableChanged(submissionID)) {
-            logger.error("Error handling submittable deleted from submission (id: {})", submissionID);
-        } else {
-            logger.trace("Triggering chained validation from submission (id: {})", submissionID);
-            chainedValidationService.triggerChainedValidation(null, submissionID);
-        }
+        fileValidationRequestHandler.handleFilesWhenSubmittableChanged(submissionID);
```

Comment:
We don't need the conditional check, anymore?
Why?

Reply:
I don't see that we needed it in the first place. The boolean returned tells us whether or not all the files in the submission had validation results. That doesn't affect whether or not I want to revalidate the other things in the submission